### PR TITLE
CI Build: update the grafana/build-container image

### PR DESCRIPTION
**Pull request by leventebalogh**
**What this PR does / why we need it**:
There is an issue started happening yesterday (2021-09-30) with the builds due to (possibly) the certificate for deb.nodesource seems to be expired. 

[Example failing pull request](https://github.com/grafana/grafana/pull/39866)
[Example failing build step](https://drone.grafana.net/grafana/grafana/35886/1/25)

[1. Related issue on github.com/nodesource](https://github.com/nodesource/distributions/issues/1266)
[2. Related issue on github.com/nodesource](https://github.com/nodesource/distributions/issues/1267)

**TL;DR**
To me it seems that there is no permanent solution to the problem yet, but one of the suggestion seemed to me like an ok temporary one. Maybe I was wrong, let me know.

### CODE REVIEW

**Comment by dsotirakis** [comment]
@leventebalogh, that's correct, but did you manage to build the image locally and then push it to dockerhub?

**Comment by dsotirakis** [comment]
Trying for ~1.5 hour now and getting some random error on `crosstool-ng` dependencies.

**Comment by leventebalogh** [comment]
> @leventebalogh, that's correct, but did you manage to build the image locally and then push it to dockerhub? And then test against it?
Hehe, unfortunately not yet, I was just shooting from the hip here 🙈 
Will try to do it as well!

**Comment by dsotirakis** [comment]
No problem, let me know how it went.
I created an issue earlier this morning, you can refer to it here https://github.com/grafana/grafana/issues/39889

**Comment by leventebalogh** [comment]
Closed this as it was actually fixed by https://github.com/grafana/grafana/pull/40002.
